packages/scenario/test/relevant.test.ts

```diff
@@ -0,0 +1,223 @@
+import {
+	bind,
+	body,
+	group,
+	head,
+	html,
+	input,
+	mainInstance,
+	model,
+	t,
+	title,
+} from '@odk-web-forms/common/test/fixtures/xform-dsl/index.ts';
+import { describe, expect, it } from 'vitest';
+import type { UntypedAnswer } from '../src/answer/UntypedAnswer.ts';
+import { Scenario } from '../src/jr/Scenario.ts';
+
+describe('Relevance - TriggerableDagTest.java', () => {
+	/**
+	 * Non-relevance is inherited from ancestor nodes, as per the W3C XForms specs:
+	 * - https://www.w3.org/TR/xforms11/#model-prop-relevant
+	 * - https://www.w3.org/community/xformsusers/wiki/XForms_2.0#The_relevant_Property
+	 */
+
+	describe('non-relevance', () => {
+		/**
+		 * **PORTING NOTES**
+		 *
+		 * - This fails because the `relevant` expressions produce node-sets, which
+		 *   will always evaluate to `true` when those nodes are present (which they
+		 *   always are in this test).
+		 *
+		 * - Those node-sets evaluate to nodes which are bound with `<bind
+		 *   type="boolean" />`, which strongly suggests that a bind's data type
+		 *   should influence casting semantics in expressions like `relevant`, and
+		 *   perhaps more generally.
+		 *
+		 * - There are some unaddressed casting considerations which **might be**
+		 *   implied by this, discussed in greater detail in porting notes on
+		 *   {@link UntypedAnswer}.
```

Comment:
Unless you feel strongly on this, I'm actually inclined to keep all of these as-is for now, and leave JavaRosa's test as-is as well.

Even if we eventually just WONTFIX these cases, it'll be clear documentation of the behavior discrepancies. But I'm also not inclined to rush towards WONTFIX: my recall is a bit fuzzy, but it feels like a common enough pattern to track.

And at least conceptually, it doesn't feel like a huge stretch for semantic expectations (even though it differs from how I'd initially/naively mapped the XPath expression semantics to the bind expression). In fact, looking back on some of the related notes on `UntypedAnswer`, I must have already been thinking along these lines (I don't think I would have framed the semantic discrepancy as "slightly different" otherwise).

Reply:
Sounds good!